WhaleSpotting/react-app/src/styles/Home.scss

```diff
@@ -0,0 +1,116 @@
+@import "./constants.scss";
+
+.home {
+    padding-bottom: $footerHeight;
+    margin-top: $headerHeight;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    background: $secondaryLight;
+    font-family: $fontFamily;
+    margin: 0;
+    padding: 0;
+}
+
+.home-contents {
+    margin-left: 6vw;
+    margin-right: 6vw;
+}
+
+.banner-image{
+    width: 100vw;
+    margin: 0;
+    height: 40vh;
+    object-fit: cover;
+}
+
+.report-button-container {
+    display: flex;
+    justify-content: center;
+    margin: 20px;
+    font-family: $fontFamily;
+
+    .report-button{
+        width: 50%;
+        background-color: $primaryDark;
+        color: $textOnPrimary;
+        border-radius: 10px;
+        border: 0px;
+        height: 50px;
+        font-family: $fontFamily;
+        
+        &:hover 
+        {
+            background-color: $primaryColour;
+        }
+    }
+}
+
+.sightings-header{
+    width: 100%;
+    display: flex;
+    justify-content: space-between;
+    font-family: $fontFamily;
+
+    .filter-button{
+        font-family: $fontFamily;
+        background-color: $primaryLight;
+        border-radius: 10px;
+        border-width: 0px;
+        color: $textOnPrimary;
+
+        &:hover 
+        {
+            background-color: $primaryCard;
```

Comment:
When it hovers to this colour, I can't see what it says, maybe make it hover to the primary colour too?

Reply:
fixed